WatchGuard patches to sources for libev-4.33 as of
Mon May  9 13:55:01 PDT 2022
The patches shown here have been applied to source .tar.gz 
files supplied with the WatchGuard Open Source Archive.

==========================================================================
--- libev-4.33/ltmain.sh.orig	2022-05-09 13:55:00.576834924 -0700
+++ libev-4.33/ltmain.sh	2022-05-09 13:55:00.684830544 -0700
@@ -8460,7 +8460,8 @@
 	      fi
 	    else
 	      # We cannot seem to hardcode it, guess we'll fake it.
-	      add_dir=-L$libdir
+	      # WG: don't fake it this introduces host pollution
+	      # add_dir=-L$libdir
 	      # Try looking first in the location we're being installed to.
 	      if test -n "$inst_prefix_dir"; then
 		case $libdir in
